CMakeLists.txt

```diff
@@ -41,6 +41,45 @@ target_include_directories(control_toolbox PUBLIC
 ament_target_dependencies(control_toolbox PUBLIC ${THIS_PACKAGE_INCLUDE_DEPENDS})
 target_compile_definitions(control_toolbox PRIVATE "CONTROL_TOOLBOX_BUILDING_LIBRARY")
 
+
+########################
+# Build control filters
+########################
+find_package(filters REQUIRED)
+find_package(generate_parameter_library REQUIRED)
+find_package(geometry_msgs REQUIRED)
+find_package(pluginlib REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(Eigen3 REQUIRED)
```

Comment:
Please add this first in the `CONTROL_FILTERS_INCLUDE_DEPENDS` list and then iterate over the list ot include all dependencies. This reduces duplication, and it makes easier maintenance of the file.